subprojects/language-groovy/src/main/java/org/gradle/api/internal/tasks/compile/GroovyCompileTransformingClassLoader.java

```diff
@@ -45,7 +45,7 @@
     }
 
     public GroovyCompileTransformingClassLoader(ClassLoader parent, ClassPath classPath) {
-        super(parent, classPath);
+        super("groovy-compile-loader", parent, classPath);
```

Comment:
I think the `transforming` is important here, I would change the name to `groovy-compile-transforming-loader`.

Reply:
:+1: